src/diffusers/loaders.py

```diff
@@ -1474,15 +1489,25 @@ def load_lora_into_text_encoder(
                 tries to not use more than 1x model size in CPU memory (including peak memory) while loading the model.
                 Only supported for PyTorch >= 1.9.0. If you are using an older version of PyTorch, setting this
                 argument to `True` will raise an error.
+            adapter_name (`str`, *optional*, defaults to `"default"`):
+                The name of the adapter to load the LoRA layers into, useful in the case of using multiple adapters
+                with the same model. Default to the default name used in PEFT library - `"default"`.
         """
         low_cpu_mem_usage = low_cpu_mem_usage if low_cpu_mem_usage is not None else _LOW_CPU_MEM_USAGE_DEFAULT
-
         # If the serialization format is new (introduced in https://github.com/huggingface/diffusers/pull/2918),
         # then the `state_dict` keys should have `self.unet_name` and/or `self.text_encoder_name` as
         # their prefixes.
         keys = list(state_dict.keys())
         prefix = cls.text_encoder_name if prefix is None else prefix
 
+        is_peft_lora_compatible = version.parse(importlib.metadata.version("transformers")) > version.parse("4.33.1")
```

Comment:
```suggestion
        is_peft_lora_compatible = version.parse(version.parse(importlib.metadata.version("transformers")).base_version) >= version.parse("4.24")
```

That's safer, in case a patch release is done for transformers 4.33 we don't accidentally include it here